src/toil/wdl/wdltoil.py

```diff
@@ -708,6 +705,94 @@ def is_url(filename: str, schemes: List[str] = ['http:', 'https:', 's3:', 'gs:',
                 return True
         return False
 
+def import_url_files(environment: WDLBindings, file_source: Union[AbstractFileStore, Toil], path: Optional[List[str]] = None, include_self: bool = False) -> Optional[Dict[str, str]]:
+    """
+    Iterate through the environment and import all files from the possible URLs. This will not import if the file is detected as local.
+    Returns a mapping of the original file names to virtualized URIs
+    :param environment: Bindings to evaluate on
+    :param file_source: Context to upload/virtualize files with
+    :param path: List of paths to search through
```

Comment:
These are for resolving relative paths, and we try them in order?

Also there are multiple things in here, so maybe it should be `paths`? Or `search_paths`?

Reply:
Changed to search_paths. The order is determined in the `search_paths` argument. The function is basically an adapted version of the previous `import_files`